lib/customer.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'order'
```

Comment:
Since you didn't actually use `Order` in this file you didn't need to import it.  This is what's causing that scary warning.  (`customer.rb` imports `order.rb` and then `order.rb` imports `customer.rb` which doesn't actually cause any problems but makes Ruby grumpy.)